lib/mapObjects/CGMarket.h

```diff
@@ -30,19 +28,19 @@ class DLL_LINKAGE IMarket
 	std::vector<EMarketMode::EMarketMode> availableModes() const;
 
 	static const IMarket *castFrom(const CGObjectInstance *obj, bool verbose = true);
-
-	template <typename Handler> void serialize(Handler &h, const int version)
-	{
-		h & o;
-	}
 };
 
 class DLL_LINKAGE CGMarket : public CGObjectInstance, public IMarket
 {
 public:
+	
+	std::set<EMarketMode::EMarketMode> marketModes;
+	int marketEfficacy = 5;
```

Comment:
Any scenarios in which this value is not loaded from config?

Reply:
it's loaded from config, here is just a default value